server/container_create_linux.go

```diff
@@ -372,6 +372,15 @@ func (s *Server) createSandboxContainer(ctx context.Context, ctr ctrIface.Contai
 	if err != nil {
 		return nil, err
 	}
+	defer func() {
+		if retErr != nil {
+			log.Infof(ctx, "createCtrLinux: deleting container %s from storage", containerInfo.ID)
```

Comment:
```suggestion
			log.Infof(ctx, "createSandboxCtr: deleting container %s from storage", containerInfo.ID)
```

Reply:
this prefix is used in all the other deferred functions in this function. I'd rather keep it here and add a follow up changing all of them if you'd prefer that

Reply:
right